### ISSUE TITLE
Deleting variations triggers an "Are You Sure?" warning, but variations are deleted

### ISSUE THREAD

**Issue by jbeales** [wp-e-commerce/WP-e-Commerce#1888]
When deleting variations by checking the boxes beside them on the Edit Product page, then choosing "Trash" from the Select box at the bottom, a message appears as if the request has failed a nonce check, (see screenshot). However, the variations are still trashed.

The same thing happens when emptying the trash - which can only be done reliably by checking the checkboxes for the variations and choosing "Delete Permanently" - the "Empty Trash" button seems to only work sometimes.

![screen shot 2015-05-25 at 9 20 56 pm](https://cloud.githubusercontent.com/assets/104935/7803734/fa4c6496-0324-11e5-8e62-409d5a561bc4.png)

**Comment by JustinSainton** [comment]
I encountered this as well. Were you able to fix it?

**Comment by jbeales** [comment]
No, but I haven't attempted to. The stuff was properly deleted and I was busy.

**Comment by misulicus** [comment]
Got this on latest master too

**Comment by misulicus** [comment]
Also its impossible to open the Trash page in the variations box
